truss/templates/server/common/truss_server.py

```diff
@@ -249,6 +249,7 @@ def create_application(self):
         def exit_self():
             # Note that this kills the current process, the worker process, not
             # the main truss_server process.
+            utils.kill_child_processes(self.pid)
```

Comment:
where does `self.pid` get assigned?

Reply:
good catch, this is a typo – should be `os.getpid()`